safe-api-koa/src/index.js

```diff
@@ -44,14 +57,18 @@ export function safeGet<I: {}, O>(
   endpoint: SafeAPI.Endpoint<I, O>,
   handler: Handler<I, O>
 ): KoaMiddleware {
-  const { url, queryParams: queryParamsRep } = extractServerData(endpoint);
-  return KoaRoute.get(url, async (context, next) => {
+  const data = extractServerData(endpoint);
+  return KoaRoute.get(data.url, async (context, ...args) => {
     const input: any = {};
+    const next = args[data.lastIndex];
```

Comment:
I'm assuming this line is the whole reason why we need `lastIndex`? why don't we just do
```js
const lastIndex = Object.keys(data.captureParamsIndex).length;
const next = args[lastIndex];
```